readme.md

```diff
@@ -5,10 +5,24 @@
   The artificial intelligence afflicted with severe depression and boredom
 </p>
 <p align="center">
+  <a href="#contributors"><img alt="All Contributors" src="https://flat.badgen.net/badge/all%20contributors/2/orange"></a>
   <a href="https://github.com/commitizen/cz-cli"><img alt="Commitizen" src="https://flat.badgen.net/badge/commitizen/friendly/green"></a>
   <a href="license"><img alt="License" src="https://flat.badgen.net/github/license/ajtorres9/marvin"></a>
 </p>
 
+## Contributors
+
+Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
+
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<!-- prettier-ignore -->
+| [<img src="https://avatars0.githubusercontent.com/u/450495?v=4" width="100px;" alt="Andrew Torres"/><br /><sub><b>Andrew Torres</b></sub>](https://andrewjtorr.es)<br />[💻](https://github.com/ajtorres9/marvin/commits?author=ajtorres9 "Code") [📖](https://github.com/ajtorres9/marvin/commits?author=ajtorres9 "Documentation") [🚇](#infra-ajtorres9 "Infrastructure (Hosting, Build-Tools, etc)") | [<img src="https://avatars0.githubusercontent.com/u/15238587?v=4" width="100px;" alt="Brian Lee"/><br /><sub><b>Brian Lee</b></sub>](https://andrewjtorr.es)<br />[📖](https://github.com/ajtorres9/marvin/commits?author=brian-dlee "Documentation") |
```

Comment:
It looks like my name is liked to your website 👀

Reply:
I need all of the publicity I can get! 😂

Actually, what I am going to do because the likeliness of someone else actually contributing to this project anytime soon is not very high, I'm going to remove you from this PR and have the bot add you so I can actually test the above badge code.